From ad32ac31952120cf20f8830233148188b6c26139 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Fri, 24 Mar 2017 05:10:15 +1300
Subject: Remove use of Module::Signature

Gentoo has its own integrity protocols anyway, and even minor
patching upsets signatures.

Its not worth the added dependency complexity.
---
 Makefile.PL      |   1 -

diff --git a/Makefile.PL b/Makefile.PL
index 0ff11dc..fa62a64 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -64,7 +64,6 @@ MSG
 }
 
 WriteMakefile(
-    (MM->can('signature_target') ? (SIGN => 1) : ()),
     NAME         => 'Net::SSH::Perl',
     DISTNAME     => 'Net-SSH-Perl',
     VERSION_FROM => 'lib/Net/SSH/Perl.pm',
-- 
2.14.3

